experimental/orchestrator/csharp_dotnetcore/03.declarative-bot/DeclarativeBotAdapter.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class DeclarativeBotAdapter : BotFrameworkHttpAdapter
+    {
+        public DeclarativeBotAdapter(ICredentialProvider credentialProvider, IConfiguration configuration, ILogger<BotFrameworkHttpAdapter> logger, IStorage storage, UserState userState, ConversationState conversationState, ResourceExplorer resourceExplorer)
+            : base(credentialProvider)
+        {
+            this.Use(new RegisterClassMiddleware<IConfiguration>(configuration));
+            this.UseStorage(storage);
+            this.UseBotState(userState);
+            this.UseBotState(conversationState);
+
+            this.OnTurnError = async (turnContext, exception) =>
```

Comment:
Are `this.` part of the samples' StyleCop configuration?

Reply:
I started with the existing adaptive samples. I'm open to changing/ updating to be consistent but we need to do it across the board. Let's file a ticket to track any code style related fixes required to this + adaptive samples?

Reply:
Filed https://github.com/microsoft/BotBuilder-Samples/issues/2681 for investigating style in .NET samples